src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/EditorTopComponent.java

```diff
@@ -190,6 +195,7 @@ public void mouseMoved(MouseEvent e) {}
         cfgLayoutButton.setSelected(diagramViewModel.getShowCFG());
         layoutButtons.add(cfgLayoutButton);
         toolBar.add(cfgLayoutButton);
+        
```

Comment:
I suggest to deactivate `HierarchicalStableLayoutManager` for difference graphs regarding
> The difference graph functionality (layout highlighting the difference between two chosen graphs) is not working for the HierarchicalStableLayoutManager.

```suggestion
        
        diagramViewModel.getGraphChangedEvent().addListener(model -> {
            // HierarchicalStableLayoutManager is not stable for difference graphs
            boolean isDiffGraph = model.getGraph().isDiffGraph();
            // deactivate HierarchicalStableLayoutManager for difference graphs
            stableSeaLayoutButton.setEnabled(!isDiffGraph);
            if (stableSeaLayoutButton.isSelected() && isDiffGraph) {
                // fallback to HierarchicalLayoutManager for difference graphs
                seaLayoutButton.setSelected(true);
            }
        });
```